### PR TITLE
feat: supportted token counting via vertexai sdk

### LINKED ISSUE #32: Migrate to the latest VertexAI SDK to use token counting via high-level API
`python-aiplatform>=1.36.0` supported token counting via high-level API.
So we don't need to use the low-level API to implement token counting.

### PR DESCRIPTION
**Pull request by adubovik**

### CODE REVIEW

**Comment by adubovik** [comment]
/deploy-review
>Deployment status: success

**Comment by adubovik** [comment]
Also it fixes #44

File: aidial_adapter_vertexai/llm/bison_adapter.py [hunk 1]
```diff
@@ -1,71 +1,170 @@
...
+    def prepare_parameters_no_stream(
+        self, params: ModelParameters
+    ) -> ChatParamsNoStream:
         return {
-            "messages": prompt.messages,
+            "max_output_tokens": params.max_tokens,
+            "temperature": params.temperature,
+            "stop_sequences": params.stop,
+            "top_p": params.top_p,
+            "candidate_count": params.n,
```
Comment (Oleksii-Klimov) [inline aidial_adapter_vertexai/llm/bison_adapter.py:68]: The method send_message_async always returns the first candidate only:
```python
yield response.text
```

Comment (adubovik) [reply aidial_adapter_vertexai/llm/bison_adapter.py:68]: Indeed, fixed.

**Review by Oleksii-Klimov — APPROVED** [review]

**Pull request merged**